conf/services/sign-in-to-hmrc-online-services-identity.cy.yml

```
@@ -8,4 +8,4 @@ contactFrontendServiceId: identity-provider-gateway
 complianceStatus: noncompliant
 statementVisibility: draft
 statementCreatedDate: 2024-03-06
-statementLastUpdatedDate: 2024-03-06
+statementLastUpdatedDate: 2024-19-02
```

Could you update this to the UK date format please?
```suggestion
statementLastUpdatedDate: 2024-02-19
```